src/init.js

```diff
@@ -87,133 +92,178 @@ export const init = async (options) => {
 		const snapfuScaffoldRepos = await fetchScaffoldRepos();
 
 		if (!snapfuScaffoldRepos?.length) {
-			console.log(chalk.red('failed to fetch scaffolds...'));
-		} else {
-			let questions = [
-				{
-					type: 'input',
-					name: 'name',
-					validate: (input) => {
-						return input && input.length > 0;
-					},
-					message: 'Please choose the name of this repository:',
-					default: path.basename(dir),
-				},
```

Comment:
Since this code has been removed, `answer.name` is undefined when applying variables to the scaffold resulting in the package.json file having some "undefined" strings in the `name` and `searchspring.siteId.{siteId}.name` properties.